lib/sound/track.cpp

```diff
@@ -181,11 +184,9 @@ void sound_ReleaseTrack(TRACK *psTrack)
 //
 void sound_CheckAllUnloaded(void)
 {
-	TRACK **currTrack;
-
-	for (currTrack = &g_apTrack[0]; currTrack != &g_apTrack[MAX_TRACKS]; ++currTrack)
+	for (auto it = g_apTrack.begin(); it != g_apTrack.end(); ++it)
```

Comment:
Here, range-based for loop can be used so that we don't have to do `(*it)` in other places.